.grunt-config/sass.js

```diff
@@ -4,13 +4,22 @@ const sass = {
 			sourcemap: true,
 			sourceMap: true
 		},
-		files: [ {
-			expand: true,
-			cwd: 'assets/dev/scss/direction',
-			src: '*.scss',
-			dest: 'assets/css',
-			ext: '.css'
-		} ]
+		files: [
+			{
+				expand: true,
+				cwd: 'assets/dev/scss/direction',
+				src: '*.scss',
+				dest: 'assets/css',
+				ext: '.css'
+			},
+			{
+				expand: true,
+				cwd: 'modules/admin-top-bar/assets/scss',
+				src: '*.scss',
+				dest: 'assets/css/modules/admin-top-bar',
+				ext: '.css',
+			},
```

Comment:
Remove. add the needed file to the `direction` folder,

Reply:
Done.